tests/testAutopas/tests/autopasInterface/AutoPasTest.cpp

```diff
@@ -152,21 +161,27 @@ TEST_F(AutoPasTest, checkRebuildingCopyCreateNew) {
 }
 
 TEST_F(AutoPasTest, checkNeedsContainerUpdate) {
-  // for linked cells this should be false
-  EXPECT_TRUE(autoPas.needsContainerUpdate());
```

Comment:
want this back :) (maybe with a true for the comment)

Reply:
Denied. This does not really make sense here.
Does this check what you want to be checked?
17e3fff149d977249c49b88b350a95f8eeba8179